services/api/src/resources/environment/helpers.ts

```diff
@@ -27,6 +27,22 @@ export const Helpers = (sqlClientPool: Pool) => {
   return {
     aliasOpenshiftToK8s,
     getEnvironmentById,
+    deleteEnvironment: async (name: string, eid: number, pid: number) => {
+      // clean up environment variables
+      await query(
+        sqlClientPool,
+        'DELETE FROM `env_vars` WHERE `environment` = :eid',
+        { eid }
+      );
```

Comment:
This looks like the same query as below. Forgot to delete after a refactor perhaps?